### PR TITLE
Omnibox refactor

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/contacts/omnibox.py [hunk 1]
```diff
@@ -18,77 +16,73 @@ def omnibox_query(org, **kwargs):
...
                 Contact,
                 search_results.contact_ids,
                 offset=0,
                 total=len(search_results.contact_ids),
                 only=("id", "uuid", "name", "org_id"),
             ).prefetch_related("org")
+        )
 
-            results += list(contacts[:per_type_limit])
-            Contact.bulk_urn_cache_initialize(contacts=results)
```
Comment (rowanseymour) [inline temba/contacts/omnibox.py:88]: this tries to run `Contact.bulk_urn_cache_initialize` on `ContactGroup` instances

**Review by norkans7 — APPROVED** [review]

**Comment by rowanseymour** [comment]
This is turning into a rewrite and the existing omnibox code works fine.. so this is getting punted to 9.3

File: temba/contacts/search/omnibox.py [hunk 2]
```diff
@@ -1,143 +0,0 @@
...
-            contacts = IDSliceQuerySet(
-                Contact,
-                search_results.contact_ids,
-                offset=0,
-                total=len(search_results.contact_ids),
-                only=("id", "uuid", "name", "org_id"),
-            ).prefetch_related("org")
-
-            results += list(contacts[:per_type_limit])
-            Contact.bulk_urn_cache_initialize(contacts=results)
```
Comment (rowanseymour) [inline temba/contacts/search/omnibox.py:88]: this tries to run `Contact.bulk_urn_cache_initialize` on `ContactGroup` instances

File: temba/contacts/views.py [hunk 3]
```diff
@@ -317,23 +317,15 @@ def create_export(self, org, user, form):
...
+            return Contact.objects.none()
 
         def render_to_response(self, context, **response_kwargs):
-            org = self.derive_org()
-            page = context["page_obj"]
-            object_list = context["object_list"]
-
-            results = omnibox_results_to_dict(org, object_list)
-
-            json_result = {"results": results, "more": page.has_next(), "total": len(results), "err": "nil"}
```
Comment (rowanseymour) [inline temba/contacts/views.py:334]: it looks like maybe this thing was page-able once upon a time but it's definitely not currently page-able.

**Pull request merged**